docs/se/guides/06_tracing.adoc

```diff
@@ -403,7 +384,7 @@ import io.opentracing.Span;
 
         Span span = request.tracer()
```

Comment:
This change is a bit dangerous.
The `request.spanContext()` may be empty (if tracing is disabled). As tracing can be disabled through configuration, this code should be modified to something like:
```
var spanBuilder = request.tracer()
  buildSpan("....");
request.spanContext().ifPresent(spanBuilder::asChildOf);
Span span = spanBuilder.start();
```